### DISCUSSION TITLE
Is it possible to access the 'players' command via this mod?

### DISCUSSION THREAD

**Discussion by forrester-marc** [LTmadness/ValheimBetterServerConfig#37] [Q&A] [answered]
It's a devcommand that sets a fixed value to override the default enemy difficulty scaling based on number of nearby players.  It would be very useful on command line servers, but in vanilla only works in peer to peer multiplayer.

**Comment by LTmadness** [comment, marked as answer]
you can use command "difficulty [nr]" to overwrite that setting
